### PR TITLE
upstream from suru-update-201703

### PR DESCRIPTION
**Pull request by mymike00**
I was unsure how to merge `debian/changelog` and it was a bit messy with mic icons @cibersheep

### CODE REVIEW

**Comment by cibersheep** [comment]
Sorry for the mic-gate :D I'll to add those after this update.
Do we need the png? I don't think we are using them in the system...

File: debian/changelog [hunk 1]
```diff
@@ -10,6 +10,13 @@ ubuntu-themes (16.11+ubports) xenial; urgency=medium
 
  -- UBports auto importer <infra@ubports.com>  Sun, 07 Jan 2018 23:13:42 +0100
 
+ubuntu-themes (16.10+17.04.20170126-0ubuntu1) zesty; urgency=medium
```
Comment (Flohack74) [inline debian/changelog:19]: The change here is OK but you need to add a new entry on top as well. It needs to increase the version number to +ubports1. This change should be done with the dch tool to keep the strict formatting. If you are unsure how to use it we can also do this.

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 1 above)
Comment (mymike00) [reply debian/changelog:19]: if you'd like to help me with that tool, I'd be glad.
otherwise you can do it by yourself.
as you prefer

**Comment by mymike00** [comment]
> Do we need the png? I don't think we are using them in the system...

will they ever be useful? if not I agree with you, but if there's a chance they'll be useful in the future and they don't waste too much space then we can keep them

**Comment by cibersheep** [comment]
I would suggest to add the png if we find we use them. I think we should use svg when possible (we already do, I think)

**Comment by NeoTheThird** [comment]
Where performance matters, pngs can be more efficiently rendered on the gpu than svgs.

**Comment by cibersheep** [comment]
Qt renders svg to the size you tell it. If you don't set the size of the svg relative to the parent width should be ok.
Space Invader was made exclusively by svg and you couldn't feel any difference :)

**Comment by mymike00** [comment]
well, let me know which is best and what to do, I'd like to merge this asap!

**Comment by cibersheep** [comment]
I would just push the svg, if we miss any png we'll solve that later (as we don't use them now).

File: debian/changelog (hunk 1 above)
Comment (mymike00) [reply debian/changelog:19]: Ok, I can do it myself (I hope) but what should I write there?
`ubuntu-themes (18.10+ubports1) xenial; urgency=medium`? is `18.10` ok?
di I need to put the name of smb before the changelog? like `[ Iain Lane ]`...
in the changelog (after `*`) what should I write? `upstream 201703 suru icons`?
in the user info? is infra@ubports.com ok or should I put my info? I think not as I didn't do quite nothing (just the PR)...
thanks

Comment (Flohack74) [reply debian/changelog:19]: Smth like this:
```
ubuntu-themes (18.10+ubports1) xenial; urgency=medium

  * Update Suru styles from upstream ....

 -- Mike <email>  <Date_exact_format>

```
Its ok to put your name and email, I mean you did fix a few things also

Comment (Flohack74) [reply debian/changelog:19]: its important to increase the version to ubports1...

Comment (mymike00) [reply debian/changelog:19]: > Its ok to put your name and email, I mean you did fix a few things also
well, no. I just took the latest files from LP and pushed here without the PNGs...

Comment (Flohack74) [reply debian/changelog:19]: Ok still put your identity. We need to bump the version and you did it so yeah ;)

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+ubuntu-themes (2019.04+ubports0) xenial; urgency=medium
```
Comment (Flohack74) [inline debian/changelog]: Please stick here with 18.10+ubports1

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 2 above)
Comment (mymike00) [reply debian/changelog]: @UniversalSuperBox told me to do like that... I'm a bit confused 😆 lol

Comment (Flohack74) [reply debian/changelog]: Really? hmmm ^^ @UniversalSuperBox explain xD

Comment (UniversalSuperBox) [reply debian/changelog]: It's a rather significant change and the versioning is arbitrary anyway. Why not make it the April 2019 update?

Comment (UniversalSuperBox) [reply debian/changelog]: Alright, I'm wrong. Please use 18.10+ubports1

**Review by Flohack74 — APPROVED** [review]

**Comment by UniversalSuperBox** [comment]
@NeoTheThird brought up a good point, this could potentially break apps which are relying on, say, the Language and Text icon being a globe. Might need to wait for OTA-10's cycle.

**Comment by mymike00** [comment]
> this could potentially break apps which are relying on, say, the Language and Text icon being a globe.

sorry, I don't understand what you're referring to

> Might need to wait for OTA-10's cycle.

I hope we can find a solution, I'd like to see this merged asap...

**Comment by UniversalSuperBox** [comment]
Wait, I think I misunderstood. Looks like icons were not normally replaced, just new ones added. Good. I'll do a final check through them.

**Review by UniversalSuperBox — APPROVED** [review]
The only icon that was replaced rather than tweaked is `preferences-desktop-locale-symbolic`. The icon is also available as `language-chooser`, which it was symlinked to before.

I think the risk that an application embedded this icon specifically as `preferences-desktop-locale-symbolic` is rather low and can be changed easily.

**Pull request merged**